apps/user-portal/src/components/multi-factor-authentication/authenticators/fido-authenticator.tsx

```diff
@@ -182,104 +302,255 @@ export const FIDOAuthenticator: React.FunctionComponent<FIDOAuthenticatorProps>
+                                                                                type="submit"
+                                                                                value={ t("common:update").toString() }
+                                                                            />
+                                                                            <Field
+                                                                                className="link-button"
+                                                                                onClick={
+                                                                                    () => {
+                                                                                        cancelEdit(
+                                                                                            device.credential
+                                                                                                .credentialId
+                                                                                        );
+                                                                                    }
+                                                                                }
+                                                                                size="small"
+                                                                                type="button"
+                                                                                value={ t("common:cancel").toString() }
+                                                                            />
+                                                                        </Form.Group>
+                                                                    </Forms>
+                                                                </List.Content>
+                                                            </List.Item>
+                                                        </Grid.Column>
+                                                        <Grid.Column width={ 2 } textAlign="right">
+                                                            <Popup
+                                                                content={ t("views:components.mfa.fido.form.remove") }
+                                                                inverted
+                                                                trigger={ (
+                                                                    <Icon
+                                                                        link={ true }
+                                                                        name="trash alternate outline"
+                                                                        color="red"
+                                                                        size="large"
```

Comment:
Can we change the icon size to small?

Reply:
The delete icon has been moved to the list and it is small now. 
![Screenshot 2020-01-13 at 14 31 58](https://user-images.githubusercontent.com/20745428/72243265-7dbb6b00-3611-11ea-9905-13d3aa487fae.png)